packages/components/src/common/avatarWrapper/AvatarWrapper.js

```diff
@@ -2,40 +2,105 @@ import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { Profile as ProfileIcon, Briefcase as BriefcaseIcon } from '@transferwise/icons';
 import Avatar from '../../avatar';
+import Badge from '../../badge';
 
 const ProfileType = {
   BUSINESS: 'BUSINESS',
   PERSONAL: 'PERSONAL',
 };
 
-export default function AvatarWrapper({ url, profileType }) {
+function OptionalBadge({ url, children, ...rest }) {
+  return url ? (
+    <Badge badge={<img src={url} alt="" />} {...rest}>
+      {children}
+    </Badge>
+  ) : (
+    children
+  );
+}
```

Comment:
Optional: Since you added the props
```suggestion
const OptionalBadge = ({ url, children, ...rest }) =>
  url ? (
    <Badge badge={<img src={url} alt="" />} {...rest}>
      {children}
    </Badge>
  ) : (
    children
  );
```